mix.exs

```diff
@@ -26,7 +26,8 @@ defmodule LiveViewNative.SwiftUI.MixProject do
       {:ex_doc, ">= 0.0.0", only: :dev, runtime: false},
       {:jason, "~> 1.2"},
       {:live_view_native_platform, "~> 0.1"},
-      {:live_view_native_stylesheet, github: "liveview-native/live_view_native_stylesheet"},
+      {:live_view_native_stylesheet, github: "NduatiK/live_view_native_stylesheet", branch: "nk-refactor-RulesParser.Helpers"},
```

Comment:
now that https://github.com/liveview-native/live_view_native_stylesheet/pull/12 is merged let's revert this LOC

Reply:
Done